dev.env

```diff
@@ -73,6 +73,17 @@ PATH=$(prepend_path "$PATH" "$VTROOT/dist/chromedriver")
 PATH=$(prepend_path "$PATH" "$VTROOT/dist/node/bin")
 export PATH
 
+# Etcd path.
```

Comment:
Currently `dev.env` does not set path for binary directory of `etcd`. That is why I added here to set the `etcd` path, so that go process can directly spawn `etcd` process.